packages/cli/src/lingui.js

```diff
@@ -19,3 +21,5 @@ program
   .command("extract [files...]", "Extracts messages from source files")
   .command("compile", "Compile message catalogs")
   .parse(process.argv)
+
+console.log(fuzzValidateCommand(program.commands, program.args))
```

Comment:
Wouldn't it print an empty line for valid cases? Might be better to check for the result before printing it.

Reply:
Thank you. I will add that check